products/sle/main.pm

```diff
@@ -266,6 +266,53 @@ if (is_update_test_repo_test && !get_var('MAINT_TEST_REPO')) {
     }
 }
 
+# This setting is used to set ADDONURL properly when SDK is required.
+# Temporary we add this on SLE 15, later it should be done in installer directly.
+# TODO: remove part for SLE 15 when not used anymore
+if (get_var('DEV_IMAGE')) {
+    my $arch    = get_var("ARCH");
+    my $build   = get_var("BUILD");
+    my $version = get_var("VERSION");
+    if (sle_version_at_least('15')) {
+        set_var('ADDONURL_SDK', "$utils::OPENQA_FTP_URL/SLE-$version-Module-Development-Tools-POOL-$arch-Build$build-Media1/");
+        set_var("ADDONURL",     "sdk");
+    }
+    elsif (sle_version_at_least('12-SP1')) {
```

Comment:
Yeah, we don't have SLE 11 in our scope and for other distris any sle_version_at_least call returns true =)